unitTests/ui/MissingTermsTest.ts

```diff
@@ -177,6 +177,46 @@ export function MissingTermsTest() {
             test.cmp.addTermForcedToAppear(specialCharacter);
             expect(test.cmp.queryStateModel.get('missingTerm')).toEqual([specialCharacter]);
           });
+
+          it(`and the term is of length 1,
+          and the term present have an apostrophe before him,
+          the term is not rendered in the page`, () => {
+            const apostropheCharacter = 's';
+            query = "efile's";
+            fakeResult.absentTerms = [apostropheCharacter];
+            test = mockComponent(query);
+            expect(test.cmp.element.childElementCount).toEqual(0);
+          });
+
+          it(`and the term is of length 1,
+          and the term present have an apostrophe after him,
+          the term is not rendered in the page`, () => {
+            const apostropheCharacter = 's';
+            query = "s'efile";
+            fakeResult.absentTerms = [apostropheCharacter];
+            test = mockComponent(query);
+            expect(test.cmp.element.childElementCount).toEqual(0);
+          });
+
+          it(`and the term is of length superior than 1,
+          and the term present have an apostrophe before him,
+          the term is rendered in the page`, () => {
+            const apostropheCharacter = 'efile';
+            query = "efile's";
+            fakeResult.absentTerms = [apostropheCharacter];
+            test = mockComponent(query);
+            expect(test.cmp.element.childElementCount).toEqual(2);
+          });
+
+          it(`and the term is of length superior than 1,
+          and the term present have an apostrophe after him,
+          the term is rendered in the page`, () => {
+            const apostropheCharacter = 'efile';
+            query = "s'efile";
+            fakeResult.absentTerms = [apostropheCharacter];
+            test = mockComponent(query);
+            expect(test.cmp.element.childElementCount).toEqual(2);
```

Comment:
What is passed as the `term` argument to the `containsApostrophe` method? Is it the `apostropheCharacter`?

Reply:
yes. this is to simulate what we receive from the API.

As I explained to you in the comments above, when a term has an `'` in it, from the API call, we will either receive what was before the `'`, what was after the `'` or both a the same time